app-frontend/src/app/services/projects/project.service.js

```diff
@@ -292,35 +292,40 @@ export default (app) => {
             ).$promise;
         }
 
-        getProjectCorners(projectId) {
-            return this.getAllProjectScenes({projectId: projectId}).then(({scenes}) => {
-                let corners = {
-                    lowerLeftLon: null,
-                    lowerLeftLat: null,
-                    upperRightLon: null,
-                    upperRightLat: null
-                };
-                scenes.forEach(scene => {
-                    let metadata = scene.sceneMetadata;
-                    if (metadata.lowerLeftCornerLatitude < corners.lowerLeftLat ||
-                        corners.lowerLeftLat === null) {
-                        corners.lowerLeftLat = metadata.lowerLeftCornerLatitude;
-                    }
-                    if (metadata.lowerLeftCornerLongitude < corners.lowerLeftLon ||
-                        corners.lowerLeftLon === null) {
-                        corners.lowerLeftLon = metadata.lowerLeftCornerLongitude;
-                    }
-                    if (metadata.upperRightCornerLatitude < corners.upperRightLat ||
-                        corners.upperRightLat === null) {
-                        corners.upperRightLat = metadata.upperRightCornerLatitude;
-                    }
-                    if (metadata.upperRightCornerLongitude < corners.upperRightLon ||
-                        corners.upperRightLon === null) {
-                        corners.upperRightLon = metadata.upperRightCornerLongitude;
-                    }
-                });
-                return corners;
-            });
+        getProjectCorners(id) {
+            // TODO Use project extent instead here
```

Comment:
This functions seems to be called in `getBounds` of `layerService`, but I don't think this `getBounds` functions is called anywhere either. I would suggest to create an issue to investigate and clean them up if needed.

Reply:
I'll make an issue